### PR TITLE
Fixes for mono

### PR DESCRIPTION
**Pull request by slide**

### CODE REVIEW

File: Build/net45.props [hunk 1]
```diff
@@ -72,7 +72,7 @@
     <Features>$(Features);FEATURE_ASYNC</Features>
     <Features>$(Features);FEATURE_GET_TYPE_INFO</Features>
     <Features>$(Features);FEATURE_DYNAMIC_EXPRESSION_VISITOR</Features>
-    <Features>$(Features);FEATURE_ASSEMBLYBUILDER_DEFINEDYNAMICASSEMBLY</Features>
+    <Features Condition=" '$(Mono)' != 'True' ">$(Features);FEATURE_ASSEMBLYBUILDER_DEFINEDYNAMICASSEMBLY</Features>
```
Comment (slozier) [inline Build/net45.props]: Why exclude this? It should be available in Mono.

Comment (slide) [inline Build/net45.props]: It was not building for me under 4.8

Comment (slozier) [inline Build/net45.props]: Odd. Ah well, it's just a switch between two APIs that are equivalent anyway, we could probably completely remove it from net45.props.

**Comment by slide** [comment]
I'll double check to make sure my codebase wasn't messed up somehow

File: Build/net45.props (hunk 1 above)
Comment (slide) [inline Build/net45.props]: I'll double check to make sure my codebase wasn't messed up somehow

**Pull request closed without merging**